src/sys/unix.rs

```diff
@@ -754,6 +740,7 @@ pub(crate) fn recv_from_vectored(
 
 /// Returns the (bytes received, sending address len, `RecvFlags`).
 #[cfg(not(target_os = "redox"))]
+#[allow(clippy::unnecessary_cast)] // For `IovLen::MAX as usize` (Already `usize` on Linux).
```

Comment:
or can simply remove the cast: `IovLen::MAX as usize`
IovLen::MAX is usize, no reason to cast it.